src/modules/CRUD/services/features.js

```diff
@@ -1,7 +1,7 @@
 import Api from '@terralego/core/modules/Api';
 
 export const fetchFeaturesList = layerId =>
-  Api.request(`crud/layer/${layerId}/features/`);
+  Api.request(`crud/layer/${layerId}/features/?page_size=2000`);
```

Comment:
Should be a param with default to 2000 ?

Reply:
As i said in the description of the PR, this is a temporary fix.

Reply:
2000 items per page.